src/emulator/modules/SceGxm/SceGxm.cpp

```diff
@@ -94,13 +103,24 @@ static bool compile_shader(GLuint shader, const GLchar *source) {
 static bool compile_shader(GLuint shader, const SceGxmProgram *program, const char *base_path) {
     GXM_PROFILE(__FUNCTION__);
 
-    const uint64_t hash = fnv1a(program, program->size);
+    char hash[65] = {};
```

Comment:
Could wrap this (and the `sha256` arg) to a struct to avoid the magic number here.